client/consts_p.go

```diff
@@ -37,6 +37,12 @@ const (
 	// operation is all but guaranteed to get them dropped.
 	UDPSleepSyncTime = time.Second
 
+	// Event log constants. These values limit the in-memory footprint
+	// of the event logging system.
+	EventLogExpiry         = 7 * 24 * time.Hour
+	EventLogLimitBytes     = 1e7
```

Comment:
This is unique to my code style but I've found it to be super useful. Only do eX in powers of three. So:

1e3, 10e3, 100e3, 1e6, 10e6, 100e6, 1e9, etc.

When you do it that way, the numbers are written the same way that they are expressed in speech. Saying 10e3 feels like a super natural replacement for "ten thousand", but 1e4 really doesn't feel as comfortable to say "ten thousand", because the leading value is 1 not 10

Reply:
so this value should be changed to 10e6

Reply:
That is super useful. I have to do mental arithmetic the other way